include/crocoddyl/core/solvers/ddp.hpp

```diff
@@ -150,20 +150,32 @@ class SolverDDP : public SolverAbstract {
    */
   virtual void allocateData();
 
+  /**
+   * @brief Return the regularization factor used to increase the damping value
+   */
+  double get_reg_incfactor() const;
```

Comment:
This is consistent with our standard for `reg_XXX` as explained above. You should notice that we do a bit differently between c++ and Python
 1. (c++) `th_stepdec`  /   (Python) `th_stepDec`
 2. (c++) `th_stepinc`  /   (Python) `th_stepInc`

You might disagree with having difference between c++ and Python.
If so, we could handle them but this will imply to change it everywhere and add a lot of deprecation functions.

I would avoid that, but you're free to tackle it in another PR.
So, I consider this discussion as closed. Please create an issue if you want to handle this.